doc/source/Resources/PyAEDTInstallerFromDesktop.py

```diff
@@ -139,22 +147,19 @@ def install_pyaedt():
                 zip_ref.extractall(unzipped_path)
 
             run_command(
-                '"{}" install --no-cache-dir --no-index --find-links={} pyaedt[all,dotnet]'.format(pip_exe, unzipped_path))
+                '"{}" install --no-cache-dir --no-index --find-links={} pyaedt[all,dotnet]'.format(pip_exe,
+                                                                                                   unzipped_path))
             run_command(
                 '"{}" install --no-cache-dir --no-index --find-links={} jupyterlab'.format(pip_exe, unzipped_path))
 
         else:
             run_command('"{}" -m pip install --upgrade pip'.format(python_exe))
             run_command('"{}" --default-timeout=1000 install wheel'.format(pip_exe))
             run_command('"{}" --default-timeout=1000 install pyaedt[all]'.format(pip_exe))
-            # run_command('"{}" --default-timeout=1000 install git+https://github.com/ansys/pyaedt.git@main'.format(pip_exe))
```

Comment:
@Samuelopez-ansys  I recommend leaving this commented line in, its helpful for testing.